### ISSUE TITLE
Consider making use of 'symfony/polyfill-intl-icu'

### ISSUE THREAD

**Issue by ocram** [delight-im/PHP-PrivacyPolicy#1]
Using [`symfony/polyfill-intl-icu`](https://github.com/symfony/polyfill-intl-icu), we could probably avoid the hard dependency on the Internationalization extension (`intl`).

**Comment by maietta** [comment]
I would recommend simply including it via composer as a dependency instead, leaving beautifully written code untainted.

**Comment by ocram** [comment]
Thanks!

Are you talking about the way the polyfill is included? Definitely wanted to do that via Composer, as `symfony/polyfill-intl-icu` is the dependency name for use with Composer.

**Comment by maietta** [comment]
Yep. Easy peasy. Keeps the project clean as not everyone needs it.
